### PR TITLE
Blogs filter

### PR DESCRIPTION
**Pull request by Zineb-Ada**
[Work in progress] This PR includes the following updates:

- Updated markdown files with the implementation of web scraping. This scraping process allowed for the addition of relevant tags to the blogs located in the '/data/planet' directory. These tags will enhance the searchability and categorization of the blogs.
- Implemented a filtering bar in the blogs section. This new feature provides users with a convenient way to filter and refine their search based on specific criteria, making it easier to find the desired content.
These changes aim to improve the overall user experience by enhancing the organization and accessibility of the blogs.

### CODE REVIEW

File: playground/src/main.ml
Comment (cuihtlauac) [inline playground/src/main.ml:1]: This should not be part of the PR. Once you've replaced by the `main` version and rebased, it should no longer pop up

**Review by cuihtlauac — CHANGES_REQUESTED** [review]

**Comment by cuihtlauac** [comment]
There is already two many origins to select the filter in a drop down menu. A possible way to go is to take inspiration from GitHub
Screenshot_2023-06-14_07-10-42
We probably want to "structure" the tags in order to distinguish origin (i.e. "from:tarides") from other, topic related, tags (s.a. "gadt", "dune" or others)

**Review by christinerose — APPROVED** [review]

File: data/planet/-hosted_on_ocamlorg/opam/signing-the-opam-repository.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ description: "Proposal for a secure distribution of OCaml packages"
 source:
   name: Opam blog
   url: ""
+  tag: "personnal blog"
```
Comment (sabine) [inline data/planet/-hosted_on_ocamlorg/opam/signing-the-opam-repository.md]: Pretty sure that the opam blog would not qualify as someone's personal blog. It's probably more of an "ocaml platform blog" blog, or an "open source project" kind of blog

**Comment by tmattio** [comment]
I understand that @Zineb-Ada won't be working on this anymore, and this doesn't seem to be in a state to be merged quickly, so I'm closing this, but if anyone would like to continue this work, a new PR will be welcome.

**Pull request closed without merging**